### DISCUSSION TITLE
Issue loading gif with ESp32S3 (AnimatedGIFpanel.ino)

### DISCUSSION THREAD

**Discussion by FM-silla** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#409] [Q&A]
Hi, i was able to output gif using the sketch AnimatedGIFpanel and ESP32S3 devkit C and a 64x32 matrix but with low brightness (its related to wrong panel initialization with FM6047 driver of p3 matrix, stillt trying to workout this) but since yesterday I cant load any gif at all. Im still able to load example sketches like "Simpletestshape" but when i try AnimatedGIFpanel doesn't work anymore. I noticed that the code was blanked on the master branch updated 8 hours ago. Maybe its something library related? 

UPDATE: Same code with ESP32-s2 works..maybe somthing strange is happening when flashing esp32s3?
UPDATED2: Setting 8MB flash and Flash mode OPI seems to solve the issue 

Do you have any ideas?

Thanks

**Comment by mrcodetastic** [comment]
It could be simply memory related. Not enough memory would lead to a black screen.

**Reply by FM-silla** [reply]
Setting 8MB flash and Flash mode OPI seems to solve the issue on ESP32s3 devkit C. The strange thing is that I didnt set these options last week. Maybe somthing related to IDE update.

I stll have the issue of low brightness with FM6047, with max current 0,3 A at brightness level 255 (all leds white). Ive tried on rpi too and i manged to work at higher brightness, then switching with ESp32s3 without powering off the matrix i was able to see gifs at higher brightness. So i think its related to initialization.

Reading the code on rpi Ive tried to modify "ESP32-HUB75-MatrixPanel-leddrivers.cpp" file in order to have
    
    gpio_set_level((gpio_num_t) _cfg.gpio.a, HIGH);  // a always on
    bool REG1[16] = {0,1,1,1,1, 1,1,1,1,1,1, 1,1,1,1,1};    //reg1 modified

but with no luck
